packages/test/mocha-test-setup/src/mochaHooks.ts

```diff
@@ -16,4 +32,9 @@ export const mochaHooks = {
         console.log = log;
         console.error = error;
     },
+    async afterAll() {
+        // Flush the logs before exiting
+        const logger: ITelemetryBufferedLogger = getTestLogger();
```

Comment:
don;t think we need this. the null logger will never need to flush, and the thing inserting a real logger should do this instead